galata/test/realtime-collaboration/notebook.test.ts

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  expect,
+  galata,
+  IJupyterLabPageFixture,
+  test
+} from '@jupyterlab/galata';
+import { User } from '@jupyterlab/services';
+import * as path from 'path';
+
+test.describe('Initialization', () => {
+  const pathUntitled = 'Untitled.ipynb';
+  const exampleNotebook = 'OutputExamples.ipynb';
+  let guestPage: IJupyterLabPageFixture;
+
+  test.beforeEach(
+    async ({
+      request,
+      appPath,
+      autoGoto,
+      baseURL,
+      browser,
+      mockSettings,
+      mockState,
+      sessions,
+      terminals,
+      tmpPath,
+      waitForApplication
+    }) => {
+      const contents = galata.newContentsHelper(request);
+      await contents.uploadFile(
+        path.resolve(
+          __dirname,
+          `../../../examples/notebooks/${exampleNotebook}`
+        ),
+        `${tmpPath}/${exampleNotebook}`
+      );
+
+      // Create a new client
+      const user: Partial<User.IUser> = {
+        identity: {
+          username: 'jovyan_2',
+          name: 'jovyan_2',
+          display_name: 'jovyan_2',
+          initials: 'JP',
+          color: 'var(--jp-collaborator-color2)'
+        }
+      };
+      guestPage = await galata.newPage(
+        appPath,
+        autoGoto,
+        baseURL!,
+        browser,
+        mockSettings,
+        mockState,
+        user,
+        sessions,
+        terminals,
+        tmpPath,
+        waitForApplication
+      );
+    }
+  );
+
+  test.afterEach(async ({ request, tmpPath }) => {
+    const contents = galata.newContentsHelper(request);
+    await contents.deleteFile(`${tmpPath}/${exampleNotebook}`);
+    // Make sure to close the page to remove the client
+    // from the awareness
+    await guestPage.close();
+  });
+
+  test('Create a notebook', async ({ page }) => {
+    // Renaming does not work
+    await page.notebook.createNew();
+    await guestPage.notebook.open(pathUntitled);
+
+    // wait for kernel to be idle
+    await guestPage.waitForTimeout(1000);
+    expect(await guestPage.screenshot()).toMatchSnapshot();
```

Comment:
You should prefer reducing the snapshot area to avoid to many update due to changes in other part of the code:

https://github.com/jupyterlab/jupyterlab/blob/60491d8185d86597d3f1882b277824635b60fc93/galata/test/jupyterlab/notebook-create.test.ts#L76-L78